### ISSUE TITLE
duplicates in list of tables

### ISSUE THREAD

**Issue by faaa6** [quarto-dev/quarto-cli#5093]
### Bug description

Hello,
I noticed that in a list of tables there are duplicates when the document is knitted to pdf and the table is spread across multiple pages.
Is there a setting or workaround to fix this?
Take care
F

### Checklist

- [ ] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [ ] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [ ] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [ ] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [ ] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by cscheid** [comment]
Thanks for the report. We need to be able to reproduce your issue with a full .qmd file that exhibits the problem. Can you provide us with that?

**Comment by faaa6** [comment]
I uploaded a sample on my git.
https://gitup.uni-potsdam.de/farntz/repo.git
This problem occurs when I use the R package "flextable"

**Comment by cscheid** [comment]
Unfortunately, I can't even run your example because with the latest tinytex and flextable, your repository fails to compile :(

```
...
! Undefined control sequence.
<argument> \Oldarrayrulewidth

l.260 ...ngth{\Oldarrayrulewidth}{\arrayrulewidth}
```

`Oldarrayrulewidth` appears to be coming from `flextable` itself, in files `R/printers.R` and `R/latex_str.R`. I don't know if there's much else we can do on our side, unfortunately.

**Comment by cscheid** [comment]
Specifically, those lines were added by this commit in `flextable`: https://github.com/davidgohel/flextable/commit/a9dcde149676398d16e5d4b39df5b03ad6c5d8f1 I don't understand the library well enough to help more on this matter.

**Comment by cderv** [comment]
> Unfortunately, I can't even run your example because with the latest tinytex and flextable

@cscheid This is an issue with dev version of **flextable** from this commit: https://github.com/davidgohel/flextable/commit/cc805164f9c3e965e9c7e65983cba42966498746

It now uses named list for `latex_dependency()` and this fails in our our processing for book (it works for single document) because of the way we are passing a list of lists to be resolved by **rmarkdown** function, which does not support that. 
https://github.com/quarto-dev/quarto-cli/blob/c56e490a0119dbb42d1e8f5a7603897465dfdc67/src/resources/rmd/rmd.R#L22

I'll open another issue  about that in our repo, but possibly this is to solve in **flextable** directly. I'll open one there too. 

Using CRAN version of **flextable** I can reproduce the issue 

> I noticed that in a list of tables there are duplicates when the document is knitted to pdf and the table is spread across multiple pages.

What @faaa6 is noticing, I can reproduce. Somehow the LaTeX table produced by **flextable** will end up having caption as heading set on each split table - and this is catched by `\listoftables` command in LaTeX. 

It is not Quarto specific as I can also reproducing using R Markdown only
````markdown
---
title: test
output: pdf_document
lot: true
---

# Chapter 01

```{r}
library(flextable)
data.frame(c(rep(1:9,5)),
           c(paste0(rep(1:9,5),"x"))) |> 
  flextable() |> 
  flextable::set_caption(caption = "A Table")
```
````

So either an issue
* in pandoc with `lot: true` and/or its PDF template
* in flextable generated LaTeX 
* An option missing somewhere in LaTeX 

Hoe it helps understand

**Comment by cderv** [comment]
Regarding the main issue here with LOT duplication, this is definitely a LaTeX behavior. 

See both 
* A knitr / R Markdown related question while using `xtable`: https://stackoverflow.com/questions/33159114/how-to-only-show-table-caption-once-in-list-of-table-for-a-table-split-onto-mu
* A pure LaTeX discussion about this: [How to use a longtable with only one entry in the list of tables](https://tex.stackexchange.com/questions/26024/how-to-use-a-longtable-with-only-one-entry-in-the-list-of-tables)

TL;DR

LaTeX using `longtable` will be split across pages when too long. Unless some specific consideration is done, the same caption will be put in head of each split table, and `\listoftables` will pick each caption. 

Not specifically clear how to tweak the LaTeX code though. 

Anyway, this is not something from Quarto directly.

**Comment by cderv** [comment]
So I opened a PR and an issue for **flextable** so that it works better. 

As I said I don't think we have an issue in Quarto as we benefit from Pandoc choices: Pandoc specifically deactivated the repetition of caption in the split table - only one at the first table, and so only one table link in the LOT.
[jgm/pandoc@ce4e4d2/src/Text/Pandoc/Writers/LaTeX/Table.hs#L53-L57](https://github.com/jgm/pandoc/blob/ce4e4d2c84f0c5c9b92f67b087fed58b760f0b05/src/Text/Pandoc/Writers/LaTeX/Table.hs#L53-L57)

So for example, no issue with a markdown table 
````markdown
---
title: test
format: 
  pdf:
    lot: true
---

# Chapter 01

| c.rep.1.9..5.. | c.paste0.rep.1.9..5....x... |
|---------------:|:----------------------------|
|              1 | 1x                          |
|              2 | 2x                          |
|              3 | 3x                          |
|              4 | 4x                          |
|              5 | 5x                          |
|              6 | 6x                          |
|              7 | 7x                          |
|              8 | 8x                          |
|              9 | 9x                          |
|              1 | 1x                          |
|              2 | 2x                          |
|              3 | 3x                          |
|              4 | 4x                          |
|              5 | 5x                          |
|              6 | 6x                          |
|              7 | 7x                          |
|              8 | 8x                          |
|              9 | 9x                          |
|              1 | 1x                          |
|              2 | 2x                          |
|              3 | 3x                          |
|              4 | 4x                          |
|              5 | 5x                          |
|              6 | 6x                          |
|              7 | 7x                          |
|              8 | 8x                          |
|              9 | 9x                          |
|              1 | 1x                          |
|              2 | 2x                          |
|              3 | 3x                          |
|              4 | 4x                          |
|              5 | 5x                          |
|              6 | 6x                          |
|              7 | 7x                          |
|              8 | 8x                          |
|              9 | 9x                          |
|              1 | 1x                          |
|              2 | 2x                          |
|              3 | 3x                          |
|              4 | 4x                          |
|              5 | 5x                          |
|              6 | 6x                          |
|              7 | 7x                          |
|              8 | 8x                          |
|              9 | 9x                          |

: Caption of the table

````

@cscheid I think we can close this issue.

**Comment by allenmanning** [comment]
Thanks for posting and the upstream debugging work.
